### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites- Edit_Screen Reader: Narrator is not reading the error message "An invalid value was given for Connection Timeout"or "Already Exists "on entering any invalid input in Connections Timeout 

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#159]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 
Screen Reader: Narrator
**MAS ID Violated:** MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Sites" in the left menu section and Press Enter.
5) Tab till "More" commands button of the existing website inside the grid and Press Enter.
6) Tab till "Edit" in the expanded menu and Press Enter.
7) Tab till "Limits" tab item and press Enter.
8) Start Screen Reader (Narrator/NVDA).
9) Verify screen reader is reading all the contents, controls, links and images properly throughout the process.

**Actual Result:**
Narrator is not reading the error message "An invalid value was given for Connection Timeout" or "Already Exists "on entering any invalid input in Connections Timeout s

**Note:** Issue is observed in every page where there is an Error Message 

**Expected Result:**
Screen Reader should read all the contents, controls, links, images properly throughout the process.

**User Impact:** 
Screen Reader users or Visually Impaired Users wont be able to understand what error they have done if Screen Reader does not read the error message properly

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug_159_Narrator is not reading the error message.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331285/Bug_159_Narrator.is.not.reading.the.error.message.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Edge; #Narrator

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
